lib/Widget/Stocks.php

```diff
@@ -392,10 +399,36 @@ public function getResource($displayId = 0)
         if (!$items = $this->getYql()) {
             return '';
         }
+        
+        if( $this->getOption('overrideTemplate') == 0 ) {
+            
+            $templates = $this->templatesAvailable();
```

Comment:
It seems like we do this a lot? Perhaps there is an opportunity to standardise access to these templates by putting methods for:

 - templatesAvailable
 - getTemplateById

Into the `ModuleWidget` class?